src/components/Wallet/NavbarBalance.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+export default class NavbarBalance extends Component {
+  static displayName = 'NavbarBalance'
+
+  static propTypes = {
+    balance: PropTypes.string,
+    currency: PropTypes.string,
+    currencyTitle: PropTypes.string,
+    initialBalance: PropTypes.string,
```

Comment:
I don't believe keeping track of `initialBalance` is necessary. Animation can occur via a react lifecycle callback and be triggered when the previous and next balance are different.